match/lib/match/storage/google_cloud_storage.rb

```diff
@@ -349,7 +349,7 @@ def check_bucket_permissions
           UI.message("\t\t 1. Click on your bucket to open it".cyan)
           UI.message("\t\t 2. Click 'Permissions'".cyan)
           UI.message("\t\t 3. Click 'Add members'".cyan)
-          UI.message("\t\t 4. Enter the email of your service account".cyan)
+          UI.message("\t\t 4. Enter the *email of your service account* (not your own email)".cyan)
```

Comment:
Isn't it obvious that service account != your own email?

Reply:
I mean, I spent hours trying to figure out what I was doing wrong but maybe I'm just dumb.